libs/SalesforceReact/SalesforceReactTests/ReactNetTests.m

```diff
@@ -22,9 +22,36 @@
  WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+#import <UIKit/UIKit.h>
+#import "ReactTestCase.h"
 
-#import "SFPluginTestSuite.h"
+@interface ReactNetTests : ReactTestCase
```

Comment:
Test for the network react bridge.
Testing everything except for getAttachment and queryMore.